# HG changeset patch
# User Dan Villiom Podlaski Christiansen <dan@cabo.dk>
# Date 1400079524 -7200
#      Wed May 14 16:58:44 2014 +0200
# Node ID 4cf89fbc0254f3ef1a4880a15aea4763b83f8242
# Parent  413007162ee5c21d11e4033369ae8ad624264b44
allow specifying default attributes and styles for tables

diff --git a/js/tinymce/plugins/table/classes/Plugin.js b/js/tinymce/plugins/table/classes/Plugin.js
--- a/js/tinymce/plugins/table/classes/Plugin.js
+++ b/js/tinymce/plugins/table/classes/Plugin.js
@@ -468,6 +468,9 @@ define("tinymce/tableplugin/Plugin", [
 			var tableElm = editor.dom.get('__mce');
 			editor.dom.setAttrib(tableElm, 'id', null);
 
+			editor.dom.setAttribs(tableElm, editor.settings.table_default_attributes || {});
+			editor.dom.setStyles(tableElm, editor.settings.table_default_styles || {});
+
 			return tableElm;
 		}
 
